modules/core/src/v2/coinFactory.ts

```diff
@@ -120,6 +122,8 @@ GlobalCoinFactory.registerCoinConstructor('zec', Zec.createInstance);
 GlobalCoinFactory.registerCoinConstructor('tzec', Tzec.createInstance);
 GlobalCoinFactory.registerCoinConstructor('algo', Algo.createInstance);
 GlobalCoinFactory.registerCoinConstructor('talgo', Talgo.createInstance);
+GlobalCoinFactory.registerCoinConstructor('trx', Trx.createInstance);
```

Comment:
Whelp, we now have a problem as the trx ticker is already taken by an erc20 token: https://github.com/BitGo/BitGoJS/blob/master/modules/core/src/config.ts#L46

Reply:
ok... this has become a topic. i've renamed this token for now, until we have a deprecation path

Reply:
i've bumped the statics version